pwa/app/components/tba/footer.tsx

```diff
@@ -0,0 +1,107 @@
+import { Link } from '@remix-run/react';
+
+import BiInfoCircleFill from '~icons/bi/info-circle-fill?width=16px&height=16px';
+
+import andymark_logo from '~/images/andymark_logo.png';
+
+export const Footer = () => {
+  // TODO: Implement this as a backend/cache thing
+  // Current time, formatted like "Aug. 23, 2024 at 11:57 AM"
+  const render_time = new Date('1/1/2000').toLocaleString('en-US', {
+    month: 'short',
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: true,
+  });
+
+  return (
+    <div className="flex flex-col space-y-3 p-5 text-center">
+      <div className="justify-center">
+        <hr className="w-full" />
+      </div>
+      <div>
+        <div className="flex flex-row justify-center">
+          <BiInfoCircleFill className="mx-1 mt-1" />
+
+          <div className="mt-0">
+            Data provided by the{' '}
```

Comment:
I don't think we actually want this on every page, only on team pages and specific event pages that are fetched from the FRC API